minikube/bootstrap.sh

```diff
@@ -208,21 +208,21 @@ set -e
 initArch
 initOS
 
-MINIKUBE_K8S_VER=v1.12.3
-MINIKUBE_BOOTSTRAPPER=kubeadm
-MINIKUBE_VM_DRIVER=virtualbox
-case "$OS" in
-  darwin)
-    # TODO: Hyperkit driver is still not stable enough. Use with later releases.
-    #MINIKUBE_VM_DRIVER=hyperkit
-    ;;
-esac
+MINIKUBE_K8S_VER=${MINIKUBE_K8S_VER:-v1.13.3}
```

Comment:
@targs08 I think `MINIKUBE_K8S_VER=v${MINIKUBE_K8S_VER:-1.13.3}` is better here for avoiding forgetting `v`.